include/dem/particle_point_line_fine_search.h

```diff
@@ -84,14 +84,12 @@ class ParticlePointLineFineSearch
    * particle-line contact force
    */
 
-  std::unordered_map<types::particle_index,
-                     particle_point_line_contact_info_struct<dim>>
+  typename dem_data_containers::dem_data_structures<
+    dim>::particle_point_line_contact_info
   particle_line_fine_search(
-    const std::unordered_map<
-      types::particle_index,
-      std::tuple<Particles::ParticleIterator<dim>, Point<dim>, Point<dim>>>
-      &           particle_line_contact_candidates,
-    const double &neighborhood_threshold);
+    const typename dem_data_containers::dem_data_structures<
+      dim>::particle_line_candidates &particle_line_contact_candidates,
+    const double &                    neighborhood_threshold);
```

Comment:
Does this need to be a reference? if it's just a double I would pass it by value.

Reply:
Corrected.